components/AddLostItem.js

```diff
@@ -48,38 +50,18 @@ export default function AddLostItem() {
         <label htmlFor="picture">
           <h4>Upload a Picture</h4>
         </label>
-        <input
+        {/* <input
           name="picture"
           type="file"
           id="picture"
           accept="image/png, image/jpeg"
           onChange={setLost.img}
-        />
+        /> */}
```

Comment:
don't include commented out code in your PRs :)

Reply:
Den brauch ich gleich noch :D aber ja das stimmt!